src/restconfig/src/test/java/org/geoserver/rest/catalog/StyleControllerTest.java

```diff
@@ -416,7 +424,7 @@ public void testPostToWorkspace() throws Exception {
         String xml = "<style>" + "<name>foo</name>" + "<filename>foo.sld</filename>" + "</style>";
         MockHttpServletResponse response =
                 postAsServletResponse(RestBaseController.ROOT_PATH + "/workspaces/gs/styles", xml);
-        assertEquals(201, response.getStatus());
```

Comment:
Is removing the assert here necessary?

Reply:
No, copy/paste error while I was adding the assertions around.